src/integration-tests/src/main/java/com/microsoft/azure/eventhubs/checkstatus/ReceiveByDateTime.java

```diff
@@ -67,82 +71,74 @@ public static void main(String[] args)
             System.exit(1);
         }
 
-        final ConnectionStringBuilder connStr = new ConnectionStringBuilder().setNamespaceName(namespaceName)
-                .setEventHubName(eventHubName).setSasKeyName(sasKeyName).setSasKey(sasKey);
+        final String connStr = String.format("Endpoint=sb://%s.servicebus.windows.net/;", namespaceName) +
+                               String.format("EntityPath=%s;", eventHubName) +
+                               String.format("SharedAccessKeyName=RootManageSharedAccessKey;", sasKeyName) +
+                               String.format("SharedAccessKey=${eventhub_key}", sasKey);
```

Comment:
Azure portal for Event Hubs provides a connection string directly.

Reply:
In this sample "namespaceName","eventHubName","sasKeyName" and "sasKey" are already set in environment. And using them to form a  connection string everywhere.